js/src/events/updates/SendUpdateModal.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+
+import {ModalForm} from '../../forms/Form'
+
+
+const EVENT_EMAIL_UPDATE_FIELDS = [
+  {name: 'subject', required: true},
+  {name: 'message', required: true, type: 'md', max_length: 255000}
+]
+const MESSAGE_NAME_TT_SLUG = 'message_to_ticket_type__'
+
+
+export class SendUpdateModal extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      initialValue: {status: props.event.status},
+      fields: this.buildFieldList()
+    }
+  }
+
+  componentDidUpdate (prevProps) {
+    if (prevProps.ticketTypes !== this.props.ticketTypes)
+      this.setState({fields: this.buildFieldList()})
```

Comment:
state should not depend on props.

You can probably just call this in `render()`?